docs/charon/charon_cli_reference.md

```diff
@@ -183,36 +194,81 @@ Usage:
+      --private-key-file string            The path to the charon enr private key file. (default ".charon/charon-enr-private-key")
+      --simnet-beacon-mock                 Enables an internal mock beacon node for running a simnet.
+      --simnet-validator-keys-dir string   The directory containing the simnet validator key shares. (default ".charon/validator_keys")
+      --simnet-validator-mock              Enables an internal mock validator client when running a simnet. Requires simnet-beacon-mock.
+      --synthetic-block-proposals          Enables additional synthetic block proposal duties. Used for testing of rare duties.
+      --validator-api-address string       Listening address (ip and port) for validator-facing traffic proxying the beacon-node API. (default "127.0.0.1:3600")
+
+```
+
+## Host a relay
+
+Relays run a libp2p circuit relay server that allows charon clusters to perform peer discovery and for charon clients behind NAT gateways to be communicated with. If you want to self-host a relay for your cluster(s) the following command will start one.
+
+```markdown
+charon relay --help
+Starts a libp2p relay that charon nodes can use to bootstrap their p2p cluster
+
+Usage:
+  charon relay [flags]
+
+Flags:
+      --auto-p2pkey                    Automatically create a p2pkey (ecdsa private key used for p2p authentication and ENR) if none found in data directory. (default true)
+      --data-dir string                The directory where charon will store all its internal data (default ".charon")
+  -h, --help                           Help for relay
+      --http-address string            Listening address (ip and port) for the relay http server serving runtime ENR. (default "127.0.0.1:3640")
+      --log-format string              Log format; console, logfmt or json (default "console")
+      --log-level string               Log level; debug, info, warn or error (default "info")
+      --loki-addresses strings         Enables sending of logfmt structured logs to these Loki log aggregation server addresses. This is in addition to normal stderr logs.
+      --loki-service string            Service label sent with logs to Loki. (default "charon")
+      --monitoring-address string      Listening address (ip and port) for the prometheus and pprof monitoring http server. (default "127.0.0.1:3620")
+      --p2p-allowlist string           Comma-separated list of CIDR subnets for allowing only certain peer connections. Example: 192.168.0.0/16 would permit connections to peers on your local network only. The default is to accept all connections.
+      --p2p-bootnode-relay             Enables using bootnodes as libp2p circuit relays. Useful if some charon nodes are not publicly accessible. Deprecated, should always be enabled. (default true)
+      --p2p-bootnodes strings          Comma-separated list of discv5 bootnode URLs or ENRs. Deprecated, use p2p-relays flag.
```

Comment:
`charon relay` is a new command, no one should have the old flags in use, we can clean out these deprecated flags if it's easy to do under the hood.

Reply:
We have decided not to remove the deprecated flags in this release but do it in the next release, ie, v0.14.0. This is to ensure we have properly tested the `charon relay` feature